tests/shell.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
This does not work for me on my NixOS machine as I seem to use a version of nixpkgs without the `rebar3WithPlugins` yet. Can you pin it to a specific version? I typically use Niv for this, but a plain nix-based pin would work as well.